docker_entrypoint.sh

```diff
@@ -148,12 +141,6 @@ if [ -e /root/.lightning/shared/lightning-rpc ]; then
 fi
 ln /root/.lightning/bitcoin/lightning-rpc /root/.lightning/shared/lightning-rpc
 
-if [ -e /root/.lightning/start9/restore.yaml ]; then
-  echo "Detected backup restore. Attempting to recover channels using emergency.recover..."
-  lightning-cli emergencyrecover
-  rm /root/.lightning/start9/restore.yaml
-fi
-
```

Comment:
You might consider removing the creation of `restore.yaml` from the `backup-restore.sh` just so it doesn't cause any problems in the future when this does get implemented. Or just keep it in mind.

Reply:
Updated